### PR TITLE
Version badges for all SDK docs

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Added version badges for all SDK docs

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by ameypv-rudder — APPROVED** [review]

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/index.mdx [hunk 1]
```diff
@@ -11,6 +11,10 @@ The **RudderStack Flutter SDK** lets you track event data from your Flutter appl
 
 Refer to the [Flutter SDK GitHub Repository](https://github.com/rudderlabs/rudder-sdk-flutter) to get a more hands-on understanding of the SDK and the related codebase.
 
+<a href="https://pub.dev/packages/rudder_sdk_flutter/versions/1.2.0" target ="_blank">
```
Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/index.mdx:14]: Here is version hardcoded.

Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/index.mdx:14]: Here is version hardcoded.

File: stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/flutter-v1.mdx [hunk 2]
```diff
@@ -10,6 +10,9 @@ description: >-
 | **IMPORTANT**: This version of the Flutter SDK will soon be deprecated. We highly recommend upgrading to the latest SDK version by going to our [**GitHub repository**](https://github.com/rudderlabs/rudder-sdk-flutter). For the latest documentation, refer to [**this**](https://rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/) guide.   |
 | :------------------------------------------------- |
 
+<a href="https://pub.dev/packages/rudder_sdk_flutter/versions/1.1.0" target ="_blank">
```
Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/flutter-v1.mdx:13]: Here is version hardcoded.

Comment (pallabmaiti) [inline stream-sources/rudderstack-sdk-integration-guides/rudderstack-flutter-sdk/flutter-v1.mdx:13]: Here is version hardcoded.

**Pull request merged**